lib/package.gd

```diff
@@ -702,70 +702,80 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##  <P/>
 ##  If the package <A>name</A> has already been loaded in a version number
-##  at least or equal to <A>version</A>, respectively,
-##  <Ref Func="LoadPackage"/> returns <K>true</K> without doing anything
-##  else.
-##  <P/>
-##  The argument <A>name</A> may be the prefix of a package name.
-##  If no package with name <A>name</A> is installed,
-##  the behaviour is as follows.
-##  If <A>name</A> is the prefix of exactly one name of an installed package
-##  then <Ref Func="LoadPackage"/> is called with this name;
-##  if the names of several installed packages start with <A>name</A> then
-##  the these names are printed, and <Ref Func="LoadPackage"/> returns
-##  <K>fail</K>.
-##  Thus the names of <E>all</E> installed packages can be shown by calling
-##  <Ref Func="LoadPackage"/> with an empty string.
+##  equal to or greater than <A>version</A>,  <Ref Func="LoadPackage"/> 
+##  returns <K>true</K> without doing anything else.
 ##  <P/>
 ##  If the optional argument <A>banner</A> is present then it must be either
 ##  <K>true</K> or <K>false</K>;
 ##  in the latter case, the effect is that no package banner is printed.
 ##  <P/>
-##  After a package has been loaded its code and documentation should be
-##  available as other parts of the &GAP; library are.
+##  After a package has been loaded, all its code and documentation becomes 
+##  available to use with the rest of the &GAP; library.
 ##  <P/>
-##  When &GAP; is started then some packages are loaded automatically.
-##  These are the packages listed in
-##  <C>GAPInfo.Dependencies.NeededOtherPackages</C> and
-##  (if this is not disabled, see below)
-##  <C>UserPreference( "PackagesToLoad" )</C>.
 ##  <P/>
-##  A &GAP; package may also install only its documentation automatically
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+DeclareGlobalFunction( "LoadPackage" );
+
+##  <#GAPDoc Label="LoadPackageAutomatic">
+##  <Subsection Label="LoadPackageAutomatic">
+##  <Heading>Automatic loading of &GAP; packages</Heading> 
+##  When &GAP; is started some packages are loaded automatically,  
+##  and these belong to two categories. 
+##  The first are those packages which are needed to start &GAP; 
+##  (at the present time, the only such package is &GAPDoc;).  
+##  Their list is contained in 
+##  <C>GAPInfo.Dependencies.NeededOtherPackages</C>. 
+##  The second are packages which are loaded during &GAP; startup by default. 
+##  The latter list may be obtained by calling 
+##  <C>UserPreference("PackagesToLoad")</C> and is customisable as described 
+##  in Section <Ref BookName="ref" Sect="Configuring User preferences"/>.
+##  <P/>
+##  While &GAP; will not start if any of the packages from the former group 
+##  is missing, loading of the packages from the latter group may be 
+##  suppressed by using the <C>-A</C> command line option 
+##  (see <Ref Sect="Command Line Options" />).
+##  <P/>
+##  A &GAP; package may also automatically install just its documentation, 
```

Comment:
Confirm here looking at https://github.com/gap-system/gap/blob/master/lib/package.gi that now we call `LoadPackageDocumentation` for all not loaded packages, so this should be rewritten accordingly.

(We may update `ValidatePackageInfo` to show warnings for obsolete components).

Reply:
After deleting this paragraph added, to the second paragraph of Ch.76, "(When &GAP; is started, <C>LoadPackageDocumentation</C> is called for all packages.)".  Note that LoadPackageDocumentation is not currently in the reference manual, but that there is XML text for it starting at line 662 of package.gd.